styles.css

```
@@ -12,32 +14,358 @@
+    color: var(--main-floating-words);
+}
+
+footer p {
+    color: var(--main-floating-words)
+}
+
+footer form {
+    color: var(--main-floating-words)
+}
+
+.inner-container__introduction a {
+    font-weight: 400;
+    letter-spacing: normal;
+}
+
+nav a {
+    font-weight: 400;
+    letter-spacing: normal;
+}
+
+a {
+    color: var(--purple);
+    font-family: Roboto, sans-serif;
+    font-weight: 600;
+    letter-spacing: .2rem;
+}
+
+p {
+    margin: 20px 0 20px 0;
+    font-family: Roboto, sans-serif;
 }
 
 /* ------------------------------- 3. Layout --------------------------------------- */
 
+header {
+    background-color: var(--purple);
+    color: var(--main-floating-words);
+    height: 60vh;
+    background-image: url("assets/squiggle_line_stripe.svg");
+    background-repeat: no-repeat;
+    background-position: 50% 110%;
+}
+
+ul {
+    list-style: none;
+}
+
+header a {
+    color: var(--main-floating-words);
+    text-decoration: none;
+}
+
+a {
+    text-decoration: none;
+}
+
+a.transition-a::after {
+    display: block;
+    content: "";
+    width: 0%;
+    height: 2px;
+    background-color: var(--purple);
+    transition: width 0.50s ease-in-out;
+}
+
+a.transition-a:hover::after {
+    width:50%;
+}
+
+
+button {
+    border-radius: 20px;
+    background-color: var(--pink);
+    color: var(--main-floating-words);
+    padding: 10px 40px;
+    border: none;
+    margin: 20px 0;
+    box-shadow: 0 8px 24px rgba(0, 0, 0, 0.40);
+}
+
+footer {
+    background: radial-gradient(circle, rgba(87,34,205,1) 39%, rgba(75,73,232,1) 93%);
+    height: 55vh;
+}
+
+.copyright{
+    background-color: var(--main-text-color) ;
+    color: var(--main-floating-words) ;
+}
+
 /* --------------- 3.1 Reusables ----------------------- */
 
+.outer-container {
+    display: flex;
+    flex-direction: row;
+    justify-content: center;
+}
+
+.inner-container {
+    flex-basis: 1200px;
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+    margin: 15px;
+
+
+}
+
+article h3 {
+    color: var(--purple);
+}
+
 /* --------------- 3.2 Other --------------------------- */
 
 /* ------------------------------- 4. Areas ---------------------------------------- */
 
 /* --------------- 4.1 Header ------------------------- */
 
+nav {
+    display: flex;
+    justify-content: flex-end;
+    padding: 20px 0 20px 0;
+}
+
+nav ul {
+    list-style: none;
+    display: flex;
+    flex-direction: row;
+    justify-content: space-around;
+    flex-basis: 600px;
+}
+
+.header-section {
+    height: calc(40vh - 60px);
+    flex-basis: 600px;
+    text-align: center;
+}
+
+.header-section span {
+    flex-basis: 100px;
+}
+
+.header-section img {
+    width: 100%;
+
+}
+
 /* --------------- 4.2 Introduction ------------------- */
+.inner-container__introduction {
+    display: flex;
+    flex-direction: column;
+    flex-basis: 700px;
+    align-items: center;
+    justify-content: space-evenly;
+    height: calc(100vh - 50vh);
```

Als het goed is is dit hetzelfde als
```suggestion
    height: 50vh;
```